src/test/java/org/apache/commons/csv/CSVPrinterTest.java

```diff
@@ -1471,4 +1471,31 @@ public void testTrimOnTwoColumns() throws IOException {
         return CSVParser.parse(expected, format).getRecords().get(0).values();
     }
 
+    @Test
+    public void testPrintReaderWithoutQuoteToWriter() throws IOException {
+        // Test to target the use of IOUtils::copyLarge.
+        // Requires the format to have no quote or escape character,
+        // value to be a java.io.Reader and the output to be a java.io.Writer.
+        final StringWriter sw = new StringWriter();
+        final String content = "testValue";
+        try (final CSVPrinter printer = new CSVPrinter(sw, CSVFormat.DEFAULT.withQuote(null))) {
+            final StringReader value = new StringReader(content);
+            printer.print(value);
+        }
+        assertEquals(content, sw.toString());
+    }
+
+    @Test
+    public void testPrintReaderWithoutQuoteToAppendable() throws IOException {
+        // Test to target the use of IOUtils::copy.
```

Comment:
Should this be a method comment (Javadoc or plain)?

Reply:
Sure. None of the other methods had Javadoc and when I initially added it as a Javadoc it was more lines than the 3 lines inside the method. It looked odd. I'll change.